xml/article_virtualization.xml

```diff
@@ -272,58 +273,61 @@
    <para>
     Standard real-time dangers apply in that if the intention is to run a
     compute intensive application with realtime priority, then the user must
-    ensure that kernel threads cannnot starve. A simple precaution is to use
-    <option>rtkthread=prio</option> and <option>rtworkqueues=prio</option>
-    kernel boot parameters, with priority set higher than anything that may
-    dominate a CPU. This is not strictly real-time capable, but it is safer
-    overall.
+    ensure that kernel threads cannot starve. A simple precaution is to use
+    the <option>rtkthread=<replaceable>PRIORITY</replaceable></option> and
+    <option>rtworkqueues=<replaceable>PRIORITY</replaceable></option> kernel
+    boot parameters. Set the value of <replaceable>PRIORITY</replaceable> to one
+    higher than any process that may dominate a CPU. This is not strictly
```

Comment:
Maybe this works?

```suggestion
    boot parameters. Set the <replaceable>PRIORITY</replaceable> values higher
    than the priority of any workload that has the potential to dominate a CPU. This is not strictly
```

Reply:
Thanks!

Included manually so that I could fix the line-wrap in the same operation.